src/Symfony/Component/Config/Definition/Builder/BooleanNodeDefinition.php

```diff
@@ -31,18 +31,6 @@ public function __construct($name, NodeParentInterface $parent = null)
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @deprecated Deprecated since version 2.8, to be removed in 3.0.
-     */
-    public function cannotBeEmpty()
```

Comment:
Wasn't this supposed to throw an exception ?  (see [this message](https://github.com/symfony/symfony/pull/15170#issuecomment-119538615))